logstash-core/lib/logstash/pipeline.rb

```diff
@@ -691,6 +691,7 @@ def collect_stats
         n.gauge(:page_capacity_in_bytes, queue.page_capacity)
         n.gauge(:max_queue_size_in_bytes, queue.max_size_in_bytes)
         n.gauge(:max_unread_events, queue.max_unread_events)
+        n.gauge(:queue_size_on_disk, queue.current_persisted_bytes)
```

Comment:
not sure I like `queue_size_on_disk` - all sizing metrics use xxx_in_bytes, I think we should stay consistent and use `queue_size_in_bytes`, WDYT?